docker-compose.yml

```diff
@@ -28,17 +28,19 @@ services:
     environment:
       - PRODUCT_ID=UNASSIGNED
       - REDIS_ENABLED=false
+      - REDIS_HOST=localhost
       - HMPPS_AUTH_EXTERNAL_URL=http://localhost:9090/auth
       - HMPPS_AUTH_URL=http://hmpps-auth:8080/auth
       # These will need to match new creds in the seed auth service auth
-      - SIGN_IN_CLIENT_ID=client-id
-      - SIGN_IN_CLIENT_SECRET=client-secret
-      - SYSTEM_CLIENT_ID=system-client-id
-      - SYSTEM_CLIENT_SECRET=system-client-secret
+      - AUTH_CODE_CLIENT_ID=hmpps-typescript-template
```

Comment:
I'm not certain this docker compose file will now work but then we don't use it for any of our local dev work or build and test pipeline